### PR TITLE
LiteXLTeam.LiteXL version 2.1.0

### PR DESCRIPTION
**Pull request by Not-a-web-Developer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62506)

### CODE REVIEW

File: manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.installer.yaml
Comment (Trenly) [inline manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.installer.yaml:10]:
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/jgmdev/lite-xl/releases/download/v2.1.0/LiteXL-2.1.0-x86_64-setup.exe
  InstallerSha256: C69ED930D96A5D3ADAB75EB0F189A1E1B4DA7C1DB7ECF393F1F9C45E8A4D30E5
  Scope: user
  InstallerSwitches: 
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/jgmdev/lite-xl/releases/download/v2.1.0/LiteXL-2.1.0-x86_64-setup.exe
  InstallerSha256: C69ED930D96A5D3ADAB75EB0F189A1E1B4DA7C1DB7ECF393F1F9C45E8A4D30E5
  Scope: machine
  InstallerSwitches: 
    Custom: /ALLUSERS
```

File: manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.installer.yaml
Comment (Trenly) [inline manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.installer.yaml:4]:
```suggestion
PackageIdentifier: jgmdev.LiteXL
```
Since this is distributed through GitHub, it is best to use the repository name as the package identifier

File: manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.locale.en-US.yaml
Comment (Trenly) [inline manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: jgmdev.LiteXL
```
Since this is distributed through GitHub, it is best to use the repository name as the package identifier

File: manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.yaml
Comment (Trenly) [inline manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.yaml:4]:
```suggestion
PackageIdentifier: jgmdev.LiteXL
```
Since this is distributed through GitHub, it is best to use the repository name as the package identifier

File: manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.locale.en-US.yaml
Comment (Trenly) [inline manifests/l/LiteXLTeam/LiteXL/2.1.0/LiteXLTeam.LiteXL.locale.en-US.yaml:10]:
```suggestion
ShortDescription: A lightweight text editor written in Lua
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Not-a-web-Developer** [comment]
I'll fix as many of these as possible right now, but lite-xl 2.1 hasn't been launched in the main repo yet, and considering the following point in the docs:
> Only one pull request per package version is allowed.

I think I should wait until jgmdev's PR is accepted for getting the winget package out. (for this, I have communicated with the core team, and they have recommended me to wait for the release to be out)

Should I close this PR? or will it be set to a draft PR until version 2.1 of lite-xl is officially released?

**Comment by Trenly** [comment]
> I'll fix as many of these as possible right now, but lite-xl 2.1 hasn't been launched in the main repo yet, and considering the following point in the docs:
> 
> > Only one pull request per package version is allowed.
> 
> I think I should wait until jgmdev's PR is accepted for getting the winget package out. (for this, I have communicated with the core team, and they have recommended me to wait for the release to be out)
> 
> Should I close this PR? or will it be set to a draft PR until version 2.1 of lite-xl is officially released?

I would change the package identifier here to `jgmdev.LiteXL` and then when the official package is released I would put it under the package identifier `LiteXL.LiteXL`

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**
